etc/entrypoint.sh

```diff
@@ -4,6 +4,8 @@ echo "Updating geoip database"
 echo "AccountID $MAXMIND_USER_ID" >> /usr/local/etc/GeoIP.conf
 echo "LicenseKey $MAXMIND_LICENSE_KEY" >> /usr/local/etc/GeoIP.conf
 geoipupdate
+echo "Make updates to the database"
+flask transform-data
```

Comment:
can we call this as a function just like the other initialization functions in main.py

Reply:
Yep, just tested and if when correctly call the function in main.py, this command is not necessary. Resolved with [f6b33a6](https://github.com/cisagov/con-pca-api/pull/839/commits/f6b33a66119c06a41763e8a4f9df2a851c9a3ca3)